compiler/src/iree/compiler/Codegen/LLVMCPU/test/select_aarch64_sve_lowering_strategy.mlir

```diff
@@ -217,3 +217,33 @@ func.func @depthwise_conv() attributes {hal.executable.target = #executable_targ
 // CHECK-SAME:     translation_info = #[[TRANSLATION]]
 // CHECK:      linalg.depthwise_conv_2d_nhwc_hwc
 // CHECK-SAME:     lowering_config = #[[CONFIG]]
+
+// -----
+
+// Regression test. SVE isn't used, but used to fail to compile when SVE was
```

Comment:
[nit] Please add a note saying why isn't SVE used. That's obvious _today_, but we might forget in the future. Thanks!

Reply:
I'm not familiar with this operation and haven't looked into it so it's actually not obvious to me 😅 I assume just because it's unsupported and we haven't looked into it?

Reply:
No worries, means the comment would be indeed helpful :)

When it comes to convs, scalable vectorisation only supports _depthwise_ convs. So that's the reason :)